docs/book/supported_features_matrix.md

```diff
@@ -15,7 +15,7 @@
 | Static PV Provisioning                                  | Yes                  | Yes              | No                 | Yes              |
 | K8s Multi-node Control Plane support                    | Yes                  | Yes              | Yes                | Yes              |
 | `WaitForFirstConsumer`                                  | Yes                  | Yes              | No                 | No               |
-
+| VolumeHealth                                  | No                  | No              | Yes (_Block PV_)                 | Yes (_Block PV_)               |
```

Comment:
Would be worth putting a note under the section `_Notes_:`, and specify minimum vSphere version requirement for this feature.